CMakeLists.txt

```diff
@@ -1176,7 +1176,7 @@ if(MINGW)
   endif()
 endif()
 
-set(public_headers include/zmq.h include/zmq_utils.h)
+set(public_headers include/zmq.h include/zmq_utils.h include/zmq_sal.h)
```

Comment:
Why is this random thing being made a public API? This makes no sense

Reply:
https://learn.microsoft.com/en-us/cpp/code-quality/understanding-sal?view=msvc-170

Reply:
Nothing observably changed, either at the API or ABI level. Annotations enforce correct usage and help reduce defects.